tests/sparse/test_basic.py

```diff
@@ -1017,22 +1018,45 @@ class TestConversion:
     def setup_method(self):
         utt.seed_rng()
 
-    @pytest.mark.skip
     def test_basic(self):
-        a = aet.as_tensor_variable(np.random.rand(5))
+        test_val = np.random.rand(5).astype(config.floatX)
+        a = aet.as_tensor_variable(test_val)
         s = csc_from_dense(a)
         val = eval_outputs([s])
-        assert str(val.dtype) == "float64"
+        assert str(val.dtype) == config.floatX
         assert val.format == "csc"
 
-    @pytest.mark.skip
-    def test_basic_1(self):
-        a = aet.as_tensor_variable(np.random.rand(5))
+        a = aet.as_tensor_variable(test_val)
         s = csr_from_dense(a)
         val = eval_outputs([s])
-        assert str(val.dtype) == "float64"
+        assert str(val.dtype) == config.floatX
         assert val.format == "csr"
 
+        test_val = np.eye(3).astype(config.floatX)
+        a = sp.sparse.csr_matrix(test_val)
+        s = as_sparse_or_tensor_variable(a)
+        res = aet.as_tensor_variable(s)
+        assert isinstance(res, SparseConstant)
+
+        a = sp.sparse.csr_matrix(test_val)
+        s = as_sparse_or_tensor_variable(a)
+        from aesara.tensor.exceptions import NotScalarConstantError
+
+        with pytest.raises(NotScalarConstantError):
+            aet.get_scalar_constant_value(s, only_process_constants=True)
+
+    # TODO:
```

Comment:
What does this test depend on? Could it become XFAIL instead or link to an issue?